src/translib/transformer/xfmr_vrf.go

```diff
@@ -259,6 +263,7 @@ var network_instance_table_name_xfmr TableXfmrFunc = func (inParams XfmrParams)
 
                 /* If keyName not expected, make it hit the sonic VRF yang to return error msg */ 
                 tblList = append(tblList, "VRF");
+                tblList = append(tblList, "VLAN")
```

Comment:
It's just an error case handling if no table is found. Don't know how it will behave with 2 table names.

Reply:
Let's please update test output for both VRF & Vlan.